docs/swagger.json

```diff
@@ -79,15 +79,14 @@
                 }
             },
             "post": {
-                "description": "Creates a condition on a server",
+                "description": "Creates a condition on a server\nSample firmwareInstall payload, response: https://github.com/metal-toolbox/conditionorc/blob/main/sample/firmwareInstall.md",
```

Comment:
This comment likely belongs in the README as an explicit pointer to developers to an example of how things work.

Reply:
Added it to the README as well. I'd like devs referring to the OpenAPI spec to find it as well.